docs/Sprint-Review/sprint-24-summary.md

```diff
@@ -0,0 +1,79 @@
+# Sprint 24 Summary
+**06/22/2021- 07/06/2021**
+
+### Updates
+ - For the week of 6/22 was impacted by Raft team having an onsite all-hands. This resulted in less productivity due to folks being out for travel and the all hands meetings, effectively rendering our sprint a little over 50% productivity.
+- NextGen XMS decisions are still blocked by having information from the NextGen Team. We have discussed moving forward with Login.gov for the time being and deprioritizing the migration efforts since we continue to be blocked.
+- The team is able to assess and move foreward on planning for ACF/AMS as of end of this sprint.
+
+
+## Sprint Goals
+
+Continue work from Sprint 23...
+Finish up ATO Functionality
+
+- Finish up file download & storage (#416, #818, #833, #834)
+- Wrap up OWASP Scan tickets (#879, #865)
+- Make decisions on upcoming releases
+- Cron Job options for release 2 (#1011)
+- NextGen XMS direction for release 1 (Epic #902, #638)
+- Document feasibility and high level estimated tasks that would be needed for ACF/AMS + Next Gen (Blocked) implementation.
+
+...Then
+Attend to Permissions for OFA Admins epic
+
+and then....
+Work on DevOPs work for Production readiness
+
+## Merged/Completed (Done/Demo, Closed)
+- Dependabot Mass Merge / Improvements [#1023](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1023)
+- As a dev/Tech Lead, I want to have automated deploys to production [#724](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/724)
+- Set up new dev environments: sandbox, raft-review, qasp-review, a11y [#848](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/848)
+
+## Submitted (QASP Review, OCIO Review)
+-[Frontend] Add a download button to the Data Files view [#416](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/416)
+- [Backend] Add endpoint to download a file from S3[#833](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/833)
```

Comment:
recommend moving into the done/demo section, since this was approved prior to demo 🚀

Reply:
done